packages/ledger/src/lib/ledger-client.ts

```diff
@@ -60,6 +60,7 @@ export interface Subscription {
 
 // Not using TransportWebHID.isSupported as it's chosen to use a Promise...
 export const isLedgerSupported = () => {
+  // @ts-ignore
```

Comment:
When you install this devDependency: `"@types/w3c-web-hid": "^1.0.2",` I think we can remove the // @ts-ignore